.github/workflows/create-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Create Release Pull Request
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        description: Type of release
+        options: ['patch', 'minor', 'major']
+        default: 'patch'
+
+jobs:
+  create_pull_request:
+    name: Create Pull Request
+    runs-on: 'ubuntu-latest'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12.18.1'
+          cache: 'yarn'
+
+      - name: Bump version
+        id: bump
+        run: |
+          yarn version --new-version ${{ github.event.inputs.version }} --no-git-tag-version
+          yarn version --cwd ./build --new-version ${{ github.event.inputs.version }} --no-git-tag-version
```

Comment:
This will work until someone updates the version manually (to re-release, for example) and forgot to update both files. But we can live with that

Reply:
I see your point, but since the goal here is to make sure there is only one path that allows you to make a release, hopefully, we'll limit the possibility of making errors by doing a manual release.